libs/partners/groq/tests/integration_tests/test_standard.py

```diff
@@ -28,11 +28,22 @@ class TestGroqLlama(BaseTestGroq):
     @property
     def chat_model_params(self) -> dict:
         return {
-            "model": "llama-3.1-70b-versatile",
+            "model": "llama-3.1-8b-instant",
```

Comment:
ooc why go smaller instead of bigger

Reply:
We can do 70b (I changed it from 8b to 70b this morning). My thought is if they support the same features we should prefer smaller models for tests in spirit of testing functionality vs. benchmarking.